File: ksmall/ksmall.c

// Starter Code
#include <stdio.h>
//#include<stdio.h>
#include <stdlib.h>

// #include <stdio.h>
#include <string.h>
// Starter Code


int smallest_value(int arr[],int n,int k)
{
    
    int k_value =-1;
    //Write your code here without removing the existing code
	//the variable 'arr' contains array of integers of size n and k is an integer.
	//modified the variable 'k_value' contain the output of the program in integer format.
    return k_value;
}


//Backend Code
int main()
{
    int *arr;
    char str[100];

    int n;
    scanf("%d", &n);
    int i = 0;

    arr = (int *)malloc(n * sizeof(int));
    int b[100];

    scanf("%s", str);
    char *token = strtok(str, "[");
    token = strtok(token, "]");
    token = strtok(token, ",");
    if (i < n)
    {
        while (token != NULL)
        {
            char *temp = token;
            int x = atoi(temp);
            *(arr + i) = x;
            token = strtok(NULL,",");
            i++;
        }
    }
    int k;
    scanf("%d",&k);
    int ans = smallest_value(arr, n,k);
    printf("%d", ans);

    return 0;
}

File: ksmall/ksmall.js
var stdin= process.openStdin();
stdin.addListener("data", function(d)
{
var x= d.toString().trim().split("\n");
let b= Number(x[0]);
let sub = JSON.parse(x[1]);
let b1= Number(x[2]);
var res= smallest_value(sub,b,k);
    console.log(res);
});




function smallest_value(arr, n,k)
{
    var k_value=-1;
    //Write your code here without removing the existing code
	//the variable 'arr' contains array of integers of size n.
	//modified the variable 'k_value' contain the output of the program in integer format.
    return k_value;
}

